data/korath/korath outfits.txt

```diff
@@ -178,6 +204,224 @@ outfit "Control Transceiver"
+	"thrusting heat" 2.9
+	"turning energy" 0.6
+	"turning heat" 1.8
+	"flare sprite" "effect/korath flare/tiny"
+		"frame rate" 8
+	"flare sound" "plasma tiny"
+	"steering flare sprite" "effect/korath flare/tiny"
+		"frame rate" 8
+	"steering flare sound" "plasma tiny"
+	description `Although the prevailing Korath attitude towards engine construction has always been with a focus on making things bigger, faster, and more powerful, the Exiles found it necessary to develop these engines to allow their fighters to carry larger weapons or more cargo, at the cost of some speed.`
+
+outfit "Bow Drive (Meteor Class)"
+	plural "Bow Drives (Meteor Class)"
+	category "Engines"
+	"cost" 187000
+	thumbnail "outfit/korath bow drive"
+	"mass" 18
+	"outfit space" -18
+	"engine capacity" -13
+	"weapon capacity" -5
+	"reverse thrust" 7.5
+	"turn" 220
+	"reverse thrusting energy" 1.3
+	"reverse thrusting heat" 2.9
+	"turning energy" 0.6
+	"turning heat" 1.8
+	"steering flare sprite" "effect/korath flare/tiny"
+		"frame rate" 8
+	"reverse flare sprite" "effect/korath flare/tiny"
+		"frame rate" 8
+	"reverse flare sound" "plasma tiny"
+	"steering flare sound" "plasma tiny"
+	description "This modification of the Meteor Class engines acts as a reverser, allowing a full range of mobility for small craft when paired with more powerful base thrusters."
+
+outfit "Reverser (Asteroid Class)"
+	plural "Reverser (Asteroid Class)"
+	category "Engines"
+	"cost" 113000
+	thumbnail "outfit/tiny korath reverser"
+	"mass" 14
+	"outfit space" -14
+	"weapon capacity" -14
+	"reverse thrust" 11.2
+	"reverse thrusting energy" 1.5
+	"reverse thrusting heat" 3.4
+	"reverse flare sprite" "effect/korath flare/tiny"
+		"frame rate" 8
+	"reverse flare sound" "plasma tiny"
+	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."
+
+outfit "Reverser (Comet Class)"
+	plural "Reversers (Comet Class)"
+	category "Engines"
+	"cost" 249000
+	thumbnail "outfit/small korath reverser"
+	"mass" 24
+	"outfit space" -24
+	"weapon capacity" -24
+	"reverse thrust" 21.8
+	"reverse thrusting energy" 2.8
+	"reverse thrusting heat" 6.5
+	"reverse flare sprite" "effect/korath flare/small"
+		"frame rate" 7.5
+	"reverse flare sound" "plasma small"
+	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."
+
+outfit "Reverser (Lunar Class)"
+	plural "Reversers (Lunar Class)"
+	category "Engines"
+	"cost" 539000
+	thumbnail "outfit/medium korath reverser"
+	"mass" 40
+	"outfit space" -40
+	"weapon capacity" -40
+	"reverse thrust" 41.2
+	"reverse thrusting energy" 4.9
+	"reverse thrusting heat" 11.9
+	"reverse flare sprite" "effect/korath flare/medium"
+		"frame rate" 7
+	"reverse flare sound" "plasma medium"
+	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."
```

Comment:
```suggestion
	description "This Korath engine has been modified for installation in a ship's weapon space, allowing it to push the ship in reverse."
	description "Dwarfing any reversers in human space, this engine provides a considerable amount of reverse thrust for any captain willing to forgo the weapons space it takes up."
```

Reply:
Colourful, though, after the comma, I might instead drive the point home by mentioning that it's commonly used on medium-sized Korath freighters

Reply:
I changed my suggestion, is it better?